src/collects/seashell/backend/dispatch.rkt

```diff
@@ -108,10 +112,21 @@
             (send-message connection `#hash((id . -4) (success . #t)
                                             (result . #hash((pid . ,pid) (test_name . ,test-name) (result . ,test-res)))))]
            [(list pid test-name "error" exit-code stderr)
+            ;; check for asan error message in "{project}/{question}/.asan.{pid}"
```

Comment:
Do you want the error message still in stderr if its invoked from runner.rkt? In dispatch.rkt I added a new filed for sanitizerMsg in the "type: done" api because I thought we could potentially separate it out from user fprintf(stderr ..) and we could do some highlight.